### ISSUE TITLE
Enable caching of static resources

### ISSUE THREAD

**Issue by sschaub** [bjucps/open-contest#78]
Every OpenContest page references several static resources, including script.js and style.css. These are not cached in the browser, and long page load times have resulted.

Add a configuration option to retain this cache-prevention behavior during development, but enable caching during a contest.

**Comment by sschaub** [comment]
During the contest, when the server is under load, the page load times increase. We tracked that down to the script.js and style.css files, which have a UUID appended to the URL by code in page.py

```
h.link(rel="stylesheet", href="/static/styles/style.css?" + uuid(), type="text/css"),
```

We can address this through a code change that generates appropriate cache control headers for development and production, or through a function that will generate a new string every 5 minutes during production.

**Comment by sschaub** [comment]
Fixed in commit 49822889930f6c203709207115645185b396349b
